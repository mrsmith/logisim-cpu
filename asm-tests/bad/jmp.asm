# no offset
jmp

# not a number
jmp xxx

# too large
jmp +128

# too small
jmp -129
