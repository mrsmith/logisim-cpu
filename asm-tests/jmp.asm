nop
nop
nop
jmp -3
# unreachable
hlt
