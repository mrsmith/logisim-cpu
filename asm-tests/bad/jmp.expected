jmp.asm:2: warn: bad jmp instruction `jmp`
jmp.asm:5: warn: bad jmp instruction `jmp xxx`
jmp.asm:8: warn: bad jmp offset `128`
jmp.asm:11: warn: bad jmp offset `-129`
